adam/language_specific/chinese/chinese_phase_1_lexicon.py

```diff
@@ -0,0 +1,140 @@
+
+We don't provide plural specifications for nouns since this isn't a
+morphologically-realized feature in Chinese.
+"""
+from adam.language_specific import (
+    FIRST_PERSON,
+    SECOND_PERSON,
+    MASS_NOUN,
+    ALLOWS_DITRANSITIVE,
+)
+from adam.language.dependency.universal_dependencies import (
+    ADJECTIVE,
+    NOUN,
+    PROPER_NOUN,
+    VERB,
+)
+from adam.language.lexicon import LexiconEntry
+from adam.language.ontology_dictionary import OntologyLexicon
+from adam.ontology.phase1_ontology import (
+    BABY,
+    BALL,
+    BIRD,
+    BLACK,
+    BLUE,
+    BOOK,
+    BOX,
+    CAR,
+    CHAIR,
+    COME,
+    COOKIE,
+    CUP,
+    DAD,
+    DOG,
+    DOOR,
+    DRINK,
+    EAT,
+    FALL,
+    FLY,
+    GAILA_PHASE_1_ONTOLOGY,
+    GIVE,
+    GO,
+    GREEN,
+    GROUND,
+    HAND,
+    HAS,
+    HAT,
+    HEAD,
+    HOUSE,
+    JUICE,
+    JUMP,
+    MILK,
+    MOM,
+    MOVE,
+    PUSH,
+    PUT,
+    RED,
+    ROLL,
+    SIT,
+    SPIN,
+    TABLE,
+    TAKE,
+    THROW,
+    TRANSPARENT,
+    TRUCK,
+    WATER,
+    WHITE,
+    LIGHT_BROWN,
+    DARK_BROWN,
+)
+
+ME = LexiconEntry(
+    "wo3",
+    NOUN,
+    plural_form="wo3 men",
+    intrinsic_morphosyntactic_properties=[FIRST_PERSON],
+)
+YOU = LexiconEntry(
+    "ni3",
+    NOUN,
+    plural_form="ni3 men",
+    intrinsic_morphosyntactic_properties=[SECOND_PERSON],
+)
+GAILA_PHASE_1_CHINESE_LEXICON = OntologyLexicon(
+    ontology=GAILA_PHASE_1_ONTOLOGY,
+    ontology_node_to_word=(
+        (GROUND, LexiconEntry("di4 myan4", NOUN)),
+        (MOM, LexiconEntry("ma1", PROPER_NOUN)),
+        (BALL, LexiconEntry("chyou2", NOUN)),
+        (TABLE, LexiconEntry("jwo1", NOUN)),
+        (PUT, LexiconEntry("fang4", VERB)),
+        (PUSH, LexiconEntry("twei1", VERB)),
+        (BOOK, LexiconEntry("shu1", NOUN)),
+        (HOUSE, LexiconEntry("wu1", NOUN)),
+        (CAR, LexiconEntry("chi4 che1", NOUN)),
+        (WATER, LexiconEntry("shwei3", NOUN, [MASS_NOUN])),
+        (JUICE, LexiconEntry("gwo3 jr1", NOUN, [MASS_NOUN])),
+        (CUP, LexiconEntry("bei1 dz", NOUN)),
+        (BOX, LexiconEntry("syang1", NOUN)),
+        (CHAIR, LexiconEntry("yi3 dz", NOUN)),
+        (HEAD, LexiconEntry("tou2", NOUN)),
+        (MILK, LexiconEntry("nyou2 nai3", NOUN, [MASS_NOUN])),
+        (HAND, LexiconEntry("shou3", NOUN)),
+        (TRUCK, LexiconEntry("ka3 che1", NOUN)),
+        (DOOR, LexiconEntry("men2", NOUN)),
+        (HAT, LexiconEntry("mau4 dz", NOUN)),
+        (COOKIE, LexiconEntry("chyu1 chi2 bing3", NOUN)),
+        (DAD, LexiconEntry("ba4", PROPER_NOUN)),
+        (BABY, LexiconEntry("bau3 bau3", NOUN)),
+        (DOG, LexiconEntry("gou3", NOUN)),
+        (BIRD, LexiconEntry("nyau3", NOUN)),
+        (GO, LexiconEntry("dzou3", VERB)),
+        (COME, LexiconEntry("lái", VERB)),
+        (TAKE, LexiconEntry("na2", VERB)),
+        (EAT, LexiconEntry("chr1", VERB)),
+        (GIVE, LexiconEntry("gei3", VERB, properties=[ALLOWS_DITRANSITIVE])),
```

Comment:
Is the di-transitive allowed in Chinese?

Reply:
From this: https://resources.allsetlearning.com/chinese/grammar/Verbs_that_take_double_objects
my understanding is yes.